### PR TITLE
Add Ceres products to SORA docs

### PR DESCRIPTION
**Pull request by MasterRadule**
Issue #223

### CODE REVIEW

File: src/ceres/overview.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+- Governance - CERES token holders can influence decisions concerning the project, either by making proposals or deciding on other’s proposals for new features.
+- Airdrops - The goddess would always generously reward her admirers. Every CERES holder can get each of three Ceres Airdrop tokens (Demeter, Apollo, and Hermes). Each airdrop will have its purpose and will be provided with initial liquidity.
+- Treasury - Part of the fees from all platforms in the Ceres ecosystem will be transferred to a public treasury wallet managed by CERES holders using the governance platform.
+- Deflationary - For the goddess to grant all the wishes of her followers, it is necessary to offer an adequate sacrifice. CERES tokens will be burned over time and the supply will constantly be reducing.
+
+<br>
+
+**CERES tokenomics**
+
+Team - 3,750 <br>
```
Comment (bgodlin) [inline src/ceres/overview.md:31]: Please provide a name for the token

**Review by bgodlin — CHANGES_REQUESTED** [review]
1. Could you specify the location where you'd prefer the pages to be included? To be done in https://github.com/sora-xor/sora-docs/blob/8ed2bedab7cb19643b5791d6e0fb911259419837/.vitepress/config.mts#L63.
2. Can you create links between the pages to improve their connections?
3. It would be great to have an image for each of the files.

File: src/ceres/demeter-farming.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+head:
+  - - meta
+    - name: description
+      content: "The Demeter Farming platform is the first of its kind on the SORA network."
+  - - meta
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.
```
Comment (WRRicht3r) [inline src/ceres/demeter-farming.md]:
```suggestion
Demeter (DEO) is the native token of the Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first self-purpose airdrop.
```

File: src/ceres/demeter-farming.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+# Demeter Farming
+
+[Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network.
```
Comment (WRRicht3r) [inline src/ceres/demeter-farming.md]:
```suggestion
The [Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network. 
```

File: src/ceres/demeter-farming.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.
+
+# Demeter Farming
+
+[Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network. 
+The unique feature of this platform is the user's ability to farm with the same liquidity in multiple pools.
```
Comment (WRRicht3r) [inline src/ceres/demeter-farming.md]:
```suggestion
This platform's unique feature is giving the user the ability to farm using the same liquidity in multiple pools. 
```

File: src/ceres/demeter-farming.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.
+
+# Demeter Farming
+
+[Demeter Farming](https://farming.deotoken.io/) platform is the first of its kind on the SORA network. 
+The unique feature of this platform is the user's ability to farm with the same liquidity in multiple pools. 
+The platform also provides Farming-as-a-Service, which means that every project can create its farming/staking pools without coding.
```
Comment (WRRicht3r) [inline src/ceres/demeter-farming.md]:
```suggestion
The platform also provides Farming-as-a-Service, meaning every project can create farming/staking pools without coding.
```

File: src/ceres/deo-arena.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+head:
+  - - meta
+    - name: description
+      content: "DEO Arena is the first Play-To-Earn game on SORA network and one of the first in DotSama ecosystem."
+  - - meta
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, web3 gaming, play-to-earn, DEO Arena, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.
```
Comment (WRRicht3r) [inline src/ceres/deo-arena.md]:
```suggestion
Demeter (DEO) is the native token of the Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first self-purpose airdrop.
```

File: src/ceres/deo-arena.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+# DEO Arena
+
+[DEO Arena](https://deoarena.io/) is the first Play-To-Earn game on SORA network and one of the first in DotSama ecosystem.
```
Comment (WRRicht3r) [inline src/ceres/deo-arena.md]:
```suggestion
[DEO Arena](https://deoarena.io/) is the first Play-To-Earn game on the SORA network and a pioneer in the DotSama ecosystem. 
```

File: src/ceres/deo-arena.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, web3 gaming, play-to-earn, DEO Arena, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first airdrop with its own purpose.
+
+# DEO Arena
+
+[DEO Arena](https://deoarena.io/) is the first Play-To-Earn game on SORA network and one of the first in DotSama ecosystem. 
+In-game currency is the XOR-native token Demeter (DEO).
```
Comment (WRRicht3r) [inline src/ceres/deo-arena.md]:
```suggestion
The in-game currency is the XOR-native token Demeter (DEO).
```

File: src/ceres/governance.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+    - name: description
+      content: "Ceres Governance platform is an important step towards decentralization"
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, governance, Polkaswap, SORA network"
+---
+
+# Ceres Governance platform
+
+[Ceres Governance platform](https://dapps.cerestoken.io/governance) is an important step towards decentralization.
```
Comment (WRRicht3r) [inline src/ceres/governance.md]:
```suggestion
The [Ceres Governance platform](https://dapps.cerestoken.io/governance) is an essential step towards decentralization. 
```

File: src/ceres/governance.md [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+      content: "Ceres Governance platform is an important step towards decentralization"
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, governance, Polkaswap, SORA network"
+---
+
+# Ceres Governance platform
+
+[Ceres Governance platform](https://dapps.cerestoken.io/governance) is an important step towards decentralization. 
+Ceres token holders are able to influence decisions concerning the project such as proposing or deciding on new feature proposals.
```
Comment (WRRicht3r) [inline src/ceres/governance.md]:
```suggestion
Ceres token holders can influence decisions concerning the project, such as proposing or implementing new features.
```

File: src/ceres/hermes-dao.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+head:
+  - - meta
+    - name: description
+      content: "Hermes DAO is a decentralized fund for investments in cryptocurrency assets."
+  - - meta
+    - name: keywords
+      content: "Ceres, Hermes, DeFi, decentralized fund, DAO, governance, Polkaswap, SORA network"
+---
+
+Hermes (HMX) is the native token of [Hermes DAO](https://hermes-dao.io/), a decentralized fund for investments in cryptocurrency assets (tokens, coins, NFTs...)
```
Comment (WRRicht3r) [inline src/ceres/hermes-dao.md]:
```suggestion
Hermes (HMX) is the native token of the [Hermes DAO](https://hermes-dao.io/), a decentralized investment fund for cryptocurrency assets (tokens, coins, NFT, etc.)
```

File: src/ceres/hermes-dao.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+managed by the Hermes DAO Parliament and the community. 
+The fund is primarily used for investments in existing crypto projects and projects in early stages.
+It is an ideal balance between a hedge fund and high risk investment venture capital with constant and stable returns. 
+
+# Hermes DAO Governance
+
+[Hermes DAO Governance](https://hermes-dao.io/governance) platform is a tool for implementing the concept of decentralization in the Hermes DAO.
```
Comment (WRRicht3r) [inline src/ceres/hermes-dao.md]:
```suggestion
The [Hermes DAO Governance](https://hermes-dao.io/governance) platform is a tool to implement the concept of decentralization in the Hermes DAO. 
```

File: src/ceres/hermes-dao.md [hunk 13]
```diff
@@ -0,0 +1,20 @@
...
+
+Hermes (HMX) is the native token of [Hermes DAO](https://hermes-dao.io/), a decentralized fund for investments in cryptocurrency assets (tokens, coins, NFTs...)
+managed by the Hermes DAO Parliament and the community. 
+The fund is primarily used for investments in existing crypto projects and projects in early stages.
+It is an ideal balance between a hedge fund and high risk investment venture capital with constant and stable returns. 
+
+# Hermes DAO Governance
+
+[Hermes DAO Governance](https://hermes-dao.io/governance) platform is a tool for implementing the concept of decentralization in the Hermes DAO. 
+Any Hermes (HMX) token holder with more than a set limit will be able to create a poll. Likewise, all voters holding Hermes (HMX) tokens above a set minimum have the same voting power. Each wallet's vote counts as one vote. This form of governance brings a greater degree of democracy.
```
Comment (WRRicht3r) [inline src/ceres/hermes-dao.md]:
```suggestion
Any Hermes (HMX) token holder above the set limit of tokens can create a poll. Likewise, all voters holding more Hermes (HMX) tokens than the specified minimum have the same voting power. Each wallet's vote counts as one vote. This form of governance brings a greater degree of democracy.
```

Comment (WRRicht3r) [reply src/ceres/hermes-dao.md]: What is the limit? How is it defined?

File: src/ceres/launchpad.md [hunk 14]
```diff
@@ -0,0 +1,16 @@
...
+    - name: description
+      content: "Ceres Launchpad is the first station for new projects on the SORA network. "
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, launchpad, Polkaswap, SORA network"
+---
+
+# Ceres Launchpad
+
+[Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network.
```
Comment (WRRicht3r) [inline src/ceres/launchpad.md]:
```suggestion
The [Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 
```

File: src/ceres/launchpad.md [hunk 15]
```diff
@@ -0,0 +1,16 @@
...
+      content: "Ceres Launchpad is the first station for new projects on the SORA network. "
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, launchpad, Polkaswap, SORA network"
+---
+
+# Ceres Launchpad
+
+[Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 
+With Launchpad, new projects can easily launch their tokens on Polkaswap and users will get more security because Ceres Liquidity Locker
```
Comment (WRRicht3r) [inline src/ceres/launchpad.md]:
```suggestion
With Launchpad, new projects can easily launch their tokens on Polkaswap, and users have added security and peace of mind thanks to the Ceres Liquidity Locker, preventing pump and dump schemes.
```

File: src/ceres/launchpad.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+and Ceres Token Locker are integrated into Launchpad.
```
Comment (WRRicht3r) [inline src/ceres/launchpad.md]:
```suggestion
The Ceres [Liquidity Locker](/liquidity-locker.md) and [Token Locker](/token-locker.md) are integrated into the Launchpad.
```

File: src/ceres/liquidity-locker.md [hunk 17]
```diff
@@ -0,0 +1,15 @@
...
+    - name: description
+      content: "Ceres Liquidity Locker is a decentralized application that brings more transparency and security to the Dotsama ecosystem."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, liquidity locker, Polkaswap, SORA network"
+---
+
+# Ceres Liquidity Locker
+
+[Ceres Liquidity Locker](https://dapps.cerestoken.io/liquidity_locker) is a decentralized application that brings more transparency and security to the Polkaswap and SORA/Polkadot ecosystem.
```
Comment (WRRicht3r) [inline src/ceres/liquidity-locker.md]:
```suggestion
The [Ceres Liquidity Locker](https://dapps.cerestoken.io/liquidity_locker) is a decentralized application that brings more transparency and security to Polkaswap and the SORA and Polkadot ecosystems.
```

File: src/ceres/liquidity-locker.md [hunk 18]
```diff
@@ -0,0 +1,15 @@
...
+Project teams are able to lock their tokens liquidity and ensure that it cannot be pulled out for some period giving investors proof of security and seriousness of the project.
```
Comment (WRRicht3r) [inline src/ceres/liquidity-locker.md]:
```suggestion
Project teams can lock their token's liquidity and ensure that it cannot be removed/drained for a determined period, giving investors proof of the project's security and legitimacy.
```

File: src/ceres/overview.md [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+    - name: description
+      content: "Ceres is developing DeFi services and utilities in DotSama ecosystem."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, liquidity locker, airdrops, demeter, hermes, apollo, DotSama, Polkaswap, SORA network"
+---
+
+## Ceres Ecosystem
+
+Ceres is one of the first DeFi service providers in the DotSama ecosystem and the first one on SORA network.
```
Comment (WRRicht3r) [inline src/ceres/overview.md]:
```suggestion
Ceres is one of the first DeFi service providers in the DotSama ecosystem and the first on the SORA network.
```

File: src/ceres/overview.md [hunk 20]
```diff
@@ -0,0 +1,48 @@
...
+Ceres provides DeFi services and utilities for the new projects and tokens in the DotSama ecosystem.<br>
```
Comment (WRRicht3r) [inline src/ceres/overview.md]:
```suggestion
Ceres provides DeFi services and utilities for new projects and tokens in the DotSama ecosystem.<br>
```

File: src/ceres/overview.md [hunk 21]
```diff
@@ -0,0 +1,48 @@
...
+CERES is a native token of the Ceres ecosystem, where the concepts of burning, airdrops, treasury and governance are in one place.
```
Comment (WRRicht3r) [inline src/ceres/overview.md]:
```suggestion
CERES is the native token of the Ceres ecosystem, where the concepts of burning, airdrops, treasury, and community governance are all in one place.
```

File: src/ceres/overview.md [hunk 22]
```diff
@@ -0,0 +1,48 @@
...
+
+## CERES token utility
+
+- Governance - CERES token holders can influence decisions concerning the project, either by making proposals or deciding on other’s proposals for new features.
+- Airdrops - The goddess would always generously reward her admirers. Every CERES holder can get each of three Ceres Airdrop tokens (Demeter, Apollo, and Hermes). Each airdrop will have its purpose and will be provided with initial liquidity.
+- Treasury - Part of the fees from all platforms in the Ceres ecosystem will be transferred to a public treasury wallet managed by CERES holders using the governance platform.
+- Deflationary - For the goddess to grant all the wishes of her followers, it is necessary to offer an adequate sacrifice. CERES tokens will be burned over time and the supply will constantly be reducing.
```
Comment (WRRicht3r) [inline src/ceres/overview.md]:
```suggestion
- Deflationary - For the goddess to grant all the wishes of her followers, it is necessary to offer an adequate sacrifice. CERES tokens will be burned over time and the supply will be constantly reducing.
```

File: src/ceres/overview.md (hunk 1 above)
Comment (WRRicht3r) [reply src/ceres/overview.md:31]: the token is called CERES

File: src/ceres/token-locker.md [hunk 23]
```diff
@@ -0,0 +1,15 @@
...
+    - name: description
+      content: "Ceres Token Locker is a decentralized application that gives project teams ability to lock their tokens."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, token locker, Polkaswap, SORA network"
+---
+
+# Ceres Token Locker
+
+[Ceres Token Locker](https://dapps.cerestoken.io/token_locker) is a decentralized application that gives project teams ability
```
Comment (WRRicht3r) [inline src/ceres/token-locker.md]:
```suggestion
The [Ceres Token Locker](https://dapps.cerestoken.io/token_locker) is a decentralized application that gives project teams the ability 
```

File: src/ceres/token-locker.md [hunk 24]
```diff
@@ -0,0 +1,15 @@
...
+      content: "Ceres Token Locker is a decentralized application that gives project teams ability to lock their tokens."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, token locker, Polkaswap, SORA network"
+---
+
+# Ceres Token Locker
+
+[Ceres Token Locker](https://dapps.cerestoken.io/token_locker) is a decentralized application that gives project teams ability 
+to lock their team/marketing/other tokens for some period in order to ensure investors that those tokens will not be sold.
```
Comment (WRRicht3r) [inline src/ceres/token-locker.md]:
```suggestion
to lock their team/marketing/other tokens for pre-determined periods to ensure investors that those tokens will not be sold.
```

File: src/ceres/tools.md [hunk 25]
```diff
@@ -0,0 +1,53 @@
...
+      content: "Ceres Tools is a platform supported by the SORA community which displays data from the Polkaswap DEX."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, tools, tokens, pairs, liquidity, prices, portfolio, swaps, charts, Polkaswap, SORA network"
+---
+
+# Ceres Tools
+
+[Ceres Tools](https://tools.cerestoken.io) is a platform supported by the SORA community which displays data from the Polkaswap DEX. 
+Users are able to track tokens prices, pairs liquidity, pool details, wallets, portfolio, swaps, charts, volumes etc.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
Users can track token prices, liquidity pairs and pool details, wallets, portfolios, swaps, charts, volumes, etc. 
```

File: src/ceres/tools.md [hunk 26]
```diff
@@ -0,0 +1,53 @@
...
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, tools, tokens, pairs, liquidity, prices, portfolio, swaps, charts, Polkaswap, SORA network"
+---
+
+# Ceres Tools
+
+[Ceres Tools](https://tools.cerestoken.io) is a platform supported by the SORA community which displays data from the Polkaswap DEX. 
+Users are able to track tokens prices, pairs liquidity, pool details, wallets, portfolio, swaps, charts, volumes etc. 
+The platform is also available in the form of a mobile application for [Android](https://play.google.com/store/apps/details?id=io.cerestoken.tools) and [iOS](https://apps.apple.com/gb/app/ceres-tools/id1590032285).
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
The platform is also available as a mobile application for the [Android](https://play.google.com/store/apps/details?id=io.cerestoken.tools) and [iOS](https://apps.apple.com/gb/app/ceres-tools/id1590032285) operating systems.
```

File: src/ceres/tools.md [hunk 27]
```diff
@@ -0,0 +1,53 @@
...
+
+
+## Tokens
+
+The prices of tokens listed on Polkaswap, their asset IDs, token locks and token holders are shown in the Tokens section.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
Token prices listed on Polkaswap, their asset IDs, token locks, and token holders are displayed in the Tokens section.
```

File: src/ceres/tools.md [hunk 28]
```diff
@@ -0,0 +1,53 @@
...
+
+## Pairs
+
+Pairs section shows liquidity, 24hr volume and pool details for each pair as
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
The pairs section shows liquidity, 24 hour volume, and pool details for each trading pair as
```

File: src/ceres/tools.md [hunk 29]
```diff
@@ -0,0 +1,53 @@
...
+well as total liquidity, total 24hr volume on Polkaswap and details about liquidity locks.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
well as total liquidity, total 24-hour volume on Polkaswap, and details about liquidity locks.
```

File: src/ceres/tools.md [hunk 30]
```diff
@@ -0,0 +1,53 @@
...
+Additionally, in this section, users can check Deposit/Withdraw liquidity events and liquidity changes history for each pair.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
In this section, users can also check Deposit/Withdrawal liquidity events and liquidity change history for each pair.
```

File: src/ceres/tools.md [hunk 31]
```diff
@@ -0,0 +1,53 @@
...
+
+## Farming
+
+Farming section shows details about all farmings and stakings on SORA in one place. Users can check Demeter, Hermes and PSWAP farmings/stakings.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
The farming section details all farming and staking on SORA in one place. Users can check Demeter, Hermes, and PSWAP farming/staking.
```

File: src/ceres/tools.md [hunk 32]
```diff
@@ -0,0 +1,53 @@
...
+
+## Burning Tracker
+
+In this section, users can track PSWAP and VAL burning.
+
+## Charts&Swaps
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
## Charts & Swaps
```

File: src/ceres/tools.md [hunk 33]
```diff
@@ -0,0 +1,53 @@
...
+
+In the Charts part of the section, users can track price movement of tokens
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
In the Charts part of the section, users can track the token price movement
```

File: src/ceres/tools.md [hunk 34]
```diff
@@ -0,0 +1,53 @@
...
+listed on Polkaswap as well as doing a Technical Analysis using various
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
listed on Polkaswap, as well as perform a Technical Analysis using various
```

File: src/ceres/tools.md [hunk 35]
```diff
@@ -0,0 +1,53 @@
...
+indicators. In the Swaps part of the section, users are able to track swaps
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
indicators. In the Swaps part of the section, users can track swaps
```

File: src/ceres/tools.md [hunk 36]
```diff
@@ -0,0 +1,53 @@
...
+(buys and sells) for each token on Ceres Tools.
+
+## Portfolio
+
+A portfolio feature allows users to track wallet’s
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
The portfolio feature allows users to track a wallets
```

File: src/ceres/tools.md [hunk 37]
```diff
@@ -0,0 +1,53 @@
...
+holdings, staking, rewards, liquidity, swaps and transfers. Also users can
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
holdings, staking, rewards, liquidity, swaps, and transfers. Users can also
```

File: src/ceres/tools.md [hunk 38]
```diff
@@ -0,0 +1,53 @@
...
+monitor the movement of the portfolio in the previous 1h, 24h, 7d and
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
monitor the portfolio's movement in 1h, 24-hour, 7-day and
```

File: src/ceres/tools.md [hunk 39]
```diff
@@ -0,0 +1,53 @@
...
+30d.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
30-day intervals.
```

File: src/ceres/tools.md [hunk 40]
```diff
@@ -0,0 +1,53 @@
...
+
+## TBC Reserves
+
+In this section, users can track TBC (Token Bonding Curve) reserves.
```
Comment (WRRicht3r) [inline src/ceres/tools.md]:
```suggestion
Users can track the SORA TBC (Token Bonding Curve) reserves in this section.
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Comment by WRRicht3r** [comment]
Please edit the config.mts page to make the Ceres content visible, thanks.

File: src/ceres/launchpad.md [hunk 41]
```diff
@@ -0,0 +1,16 @@
...
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, launchpad, Polkaswap, SORA network"
+---
+
+# Ceres Launchpad
+
+The [Ceres Launchpad](https://dapps.cerestoken.io/launchpad) is the first station for new projects on the SORA network. 
+With Launchpad, new projects can easily launch their tokens on Polkaswap, and users have added security and peace of mind thanks to the Ceres Liquidity Locker, preventing pump and dump schemes.
+The Ceres [Liquidity Locker](/liquidity-locker.md) and [Token Locker](/token-locker.md) are integrated into the Launchpad.
```
Comment (WRRicht3r) [inline src/ceres/launchpad.md]:
```suggestion
The Ceres [Liquidity Locker](./liquidity-locker.md) and [Token Locker](./token-locker.md) are integrated into the Launchpad.
```

Comment (WRRicht3r) [reply src/ceres/launchpad.md]: pls commit this urgent @DanijelCBS ! 
Cheers

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

File: src/ceres/launchpad.md (hunk 41 above)
Comment (MasterRadule) [reply src/ceres/launchpad.md]: Fixed

**Comment by MasterRadule** [comment]
> 1. Could you specify the location where you'd prefer the pages to be included? To be done in https://github.com/sora-xor/sora-docs/blob/8ed2bedab7cb19643b5791d6e0fb911259419837/.vitepress/config.mts#L63
>    .
> 2. Can you create links between the pages to improve their connections?
> 3. It would be great to have an image for each of the files.


1. Done
2. Done
3. Done

File: src/ceres/hermes-dao.md [hunk 42]
```diff
@@ -0,0 +1,22 @@
...
+title: "Hermes DAO | SORA Docs"
+head:
+  - - meta
+    - name: description
+      content: "Hermes DAO is a decentralized fund for investments in cryptocurrency assets."
+  - - meta
+    - name: keywords
+      content: "Ceres, Hermes, DeFi, decentralized fund, DAO, governance, Polkaswap, SORA network"
+---
+
```
Comment (WRRicht3r) [inline src/ceres/hermes-dao.md:11]:
```suggestion
# Hermes DAO 
```

File: src/ceres/demeter-farming.md [hunk 43]
```diff
@@ -0,0 +1,20 @@
...
+title: "Demeter Farming | SORA Docs"
+head:
+  - - meta
+    - name: description
+      content: "The Demeter Farming platform is the first of its kind on the SORA network."
+  - - meta
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, farming, staking, farming-as-a-service, Polkaswap, SORA network"
+---
+
```
Comment (WRRicht3r) [inline src/ceres/demeter-farming.md:11]:
```suggestion
# Demeter (DEO)
```

File: src/ceres/deo-arena.md [hunk 44]
```diff
@@ -0,0 +1,21 @@
...
+head:
+  - - meta
+    - name: description
+      content: "DEO Arena is the first Play-To-Earn game on SORA network and one of the first in DotSama ecosystem."
+  - - meta
+    - name: keywords
+      content: "Ceres, Demeter, DeFi, web3 gaming, play-to-earn, DEO Arena, Polkaswap, SORA network"
+---
+
+Demeter (DEO) is the native token of the Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first self-purpose airdrop.
```
Comment (WRRicht3r) [inline src/ceres/deo-arena.md]:
```suggestion

```

File: src/ceres/deo-arena.md [hunk 45]
```diff
@@ -0,0 +1,21 @@
...
+
+# DEO Arena
+
+[DEO Arena](https://deoarena.io/) is the first Play-To-Earn game on the SORA network and a pioneer in the DotSama ecosystem.
+The in-game currency is the XOR-native token Demeter (DEO).
+Players spawn in the arena and the goal is to eliminate opponents using different weapons.
+Each player enters the match with a certain amount of DEO tokens, which are taken by the winner at the end.
+
```
Comment (WRRicht3r) [inline src/ceres/deo-arena.md:18]:
```suggestion
## Demeter Token (DEO)
Demeter (DEO) is the native token of the Demeter farming platform and DEO Arena Web3 game. In the Ceres ecosystem, Demeter is the first self-purpose airdrop.
```

File: src/ceres/overview.md [hunk 46]
```diff
@@ -0,0 +1,50 @@
...
+head:
+  - - meta
+    - name: description
+      content: "Ceres is developing DeFi services and utilities in DotSama ecosystem."
+  - - meta
+    - name: keywords
+      content: "Ceres, DeFi, liquidity locker, airdrops, demeter, hermes, apollo, DotSama, Polkaswap, SORA network"
+---
+
+## Ceres Ecosystem
```
Comment (WRRicht3r) [inline src/ceres/overview.md]:
```suggestion
# Ceres Ecosystem Overview
```

**Review by WRRicht3r — CHANGES_REQUESTED** [review]

**Review by WRRicht3r — APPROVED** [review]

**Review by bgodlin — APPROVED** [review]

**Pull request merged**